### ISSUE TITLE
Fix: Full NodeJS Path: Rewrite headings to follow sentence case Style Guide

### ISSUE THREAD

**Issue by ManonLef** [TheOdinProject/curriculum#25861]
### Describe your suggestion

Our [Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#headings) requires all headings to be in **sentence case**. This means many lessons need to be reviewed and corrected as needed.

I would like to open this up for **first-time contributors or those still very new to contributing** to work on. You can follow the links to see how big the sections are. Some will have only a few lessons and some will have quite a lot more. Feel free to pick one you feel comfortable with. 

Some sections might have been corrected already but it's good to review them to make sure nothing is missed. 

Example:

```md
### Knowledge Check

should be changed to

### Knowledge check
```

## Guidance:

- Comment on this issue to let us know you'd like to get a section assigned to you. You can also ask for a specific one
- Please wait until you are assigned by a maintainer before starting to work on this issue to avoid multiple people working on the same lessons
- Make sure to read the [contributing guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md) before starting your first contribution
- Please add the following to your PR in the Issues section: `Related to #25861` , since this issue is divided into sections, individual PR's should **NOT INCLUDE** the `closes` issue link. 

## Progress and assignees table

| Section(s) | Assignee | completed |
| ------------- | ------------- | ------------- |
| **Intermediate HTML and CSS Course** | ------------- | ------------- |
|  |  |  |
| [Intermediate HTML and CSS](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css#intermediate-html-concepts)  && [Intermediate CSS Concepts](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css#intermediate-css-concepts) | @amc131 | ✅ https://github.com/TheOdinProject/curriculum/pull/25894 |
| [Forms](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css#forms) && [Grid](https://www.theodinproject.com/paths/full-stack-javascript/courses/intermediate-html-and-css#grid) | @kristenmazza | ✅  https://github.com/TheOdinProject/curriculum/pull/25866 |
| **JavaScript Course** | ------------- | ------------- |
||||
| [Introduction](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#introduction) && [Organizing your JavaScript Code](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#organizing-your-javascript-code) | @fabulousgk | ✅  https://github.com/TheOdinProject/curriculum/pull/25864 |
| [JavaScript in the Real World](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#javascript-in-the-real-world) && [Asynchronous JavaScript and APIs](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#asynchronous-javascript-and-apis)| @therealseanwallace | ✅ https://github.com/TheOdinProject/curriculum/pull/25879 |
| [A Bit of Computer Science](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#a-bit-of-computer-science) | @hmzq-404 | ✅  https://github.com/TheOdinProject/curriculum/pull/25880 |
| [Testing JS](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#testing-javascript) && [Intermediate Git](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#intermediate-git)| @fabulousgk  | ✅ https://github.com/TheOdinProject/curriculum/pull/25957 |
| **Advanced HTML and CSS** | ------------- | ------------- |
||||
| [All Lessons](https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css) (has recently been updated so little to no work) | @Hoslok-1  | ✅ https://github.com/TheOdinProject/curriculum/pull/25910 |
| **React (NEW)** | ------------- | ------------- |
||||
| [Intro](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#introduction) && [Getting Started](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#getting-started-with-react) | @OmarNaguib | ✅ https://github.com/TheOdinProject/curriculum/pull/25865 |
| [States and Effects](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#states-and-effects) && [Class Components](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#class-components) | @RazvanBugoi  | ✅ https://github.com/TheOdinProject/curriculum/pull/25932 |
| [React Testing](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#react-testing) && [The React Ecosystem](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#the-react-ecocsystem)| @Babb2000 | ✅ https://github.com/TheOdinProject/curriculum/pull/25896 |
| [More React Concepts](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#more-react-concepts) && [React and the Backend](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#react-and-the-backend) && [Conclusion](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#react-and-the-backend)| @JustWaveThings  | ✅ https://github.com/TheOdinProject/curriculum/pull/25886 |
| **NodeJS** | ------------- | ------------- |
||||
| [Introduction to NodeJS](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs#introduction-to-nodejs) && [MongoDB](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs#mongodb)| @YamamJamal  | ✅ https://github.com/TheOdinProject/curriculum/pull/25945 |
|[Express & Mongoose](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs#express-mongoose) && [Authentication](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs#authentication) && [APIs](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs#apis) | @F4OST | ✅ https://github.com/TheOdinProject/curriculum/pull/25892 https://github.com/TheOdinProject/curriculum/pull/25893 | 
| [Testing Express](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs#testing-express) && [Full Stack Projects](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs#full-stack-projects) && [FINAL PROJECT](https://www.theodinproject.com/paths/full-stack-javascript/courses/nodejs#final-project) | @Hoslok-1 | ✅ https://github.com/TheOdinProject/curriculum/pull/25910 |
| **Getting Hired** | ------------- | ------------- |
||||
| [Preparing for Your Job Search](https://www.theodinproject.com/paths/full-stack-javascript/courses/getting-hired#preparing-for-your-job-search) | @aaronlmjohnson | ✅ https://github.com/TheOdinProject/curriculum/pull/25863 |
|[Applying to and Interviewing for Jobs](https://www.theodinproject.com/paths/full-stack-javascript/courses/getting-hired#applying-to-and-interviewing-for-jobs) | @latsonj | ✅ https://github.com/TheOdinProject/curriculum/pull/25907 | 

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/paths/full-stack-javascript

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### (Optional) Discord Name

ManonL (feel free to tag me there in public if you need help while assigned here)

**Comment by YamamJamal** [comment]
Hello, I am a student getting a degree in computer science and in an open source course and looking to contribute to an open source project for the first time. I think this would be appropriate for me since I this would be my very first time contributing! I would like to work on the Introduction to NodeJS && MongoDB section. It would be very much appreciated and thank you!

**Comment by ManonLef** [comment]
You've got it @YamamJamal ! Thank you for doing this

**Comment by hmzq-404** [comment]
Can I work on the `A bit of Computer Science` lessons?

**Comment by ManonLef** [comment]
Absolutely @hmzq-404 ! Thank you!

**Comment by amc131** [comment]
I've made a small contribution before, but I can help as well unless you have enough first time contributors

**Comment by ManonLef** [comment]
@amc131 this is open to fairly new contributors as well. Any preference or would you like me to put your name on the first one in the table?

**Comment by amc131** [comment]
@ManonLef No preference, I can help with any!

**Comment by aaronlmjohnson** [comment]
Hi, I'm looking to make some first time contributions. Could I work on the "Preparing for Your Job Search" section?

**Comment by ManonLef** [comment]
@aaronlmjohnson all yours!

@amc131 I added you as well in the table! 

Thanks you both 🥳

**Comment by OmarNaguib** [comment]
Would be my second contribution, first one was 2 minutes ago :D .
Would like to help

**Comment by ManonLef** [comment]
@OmarNaguib that's wonderful! Any preference or would you like to be assigned a random one?

**Comment by OmarNaguib** [comment]
first one is good

**Comment by fabulousgk** [comment]
Hello, I would do one of these as well.  Just assign me one please!

**Comment by ManonLef** [comment]
ypu've got it @fabulousgk !

**Comment by kristenmazza** [comment]
I just made my first contribution with the foundation headings and would love some more practice. I'm happy with any section available.

**Comment by ManonLef** [comment]
Added you @kristenmazza ☺️ thank you!

**Comment by therealseanwallace** [comment]
I'd  be happy to help out with some of this. I don't mind which part I work on, feel free to just assign me something if you like @ManonLef.

**Comment by ManonLef** [comment]
Thanks for helping out here @therealseanwallace !

**Comment by fabulousgk** [comment]
@ManonLef  couple of things I need some guidance on...

1) Introduction/How this course will work -> The Path  - Is Path a name that should be cap?
2) Are the names of patterns proper and therefore cap, eg Module pattern or module pattern

Thank you

**Comment by KadeKorn** [comment]
Hello, I would like to contribute as well. Please let me know how I can help

**Comment by kristenmazza** [comment]
@ManonLef Is there a guideline on whether to capitalize grid and flex/flexbox as proper nouns in the section headings? I previously updated the Flexbox section and used lowercase because the lesson text used lowercase. However, now I am updating the Grid section and I am finding both grid and flex/flexbox are capitalized in the lesson text.

**Comment by ManonLef** [comment]
@fabulousgk I don't think any of those need to be capitalized officially. 

@kristenmazza both `grid` and `flex` don't need to be capitalized. If they do it in the lesson that might need an issue of its own :D. IF it's only a few instances, feel free to include those if you want.

**Comment by ManonLef** [comment]
@KadeKorn thank you, I will add you to the table!

**Comment by OmarNaguib** [comment]
I made a few judgment calls,
1- React ,proper noun
2-Let’s start Reactin’, because of 1
3-Create React App, proper noun? they have it that way on their website.

onceT you approve i can make the PR

**Comment by ManonLef** [comment]
@OmarNaguib sounds good to me!

**Comment by kristenmazza** [comment]
> @kristenmazza both `grid` and `flex` don't need to be capitalized. If they do it in the lesson that might need an issue of its own :D. IF it's only a few instances, feel free to include those if you want.

Sounds good! I will leave it for another issue, as there are many instances throughout the grid lessons.

**Comment by latsonj** [comment]
Can I be assigned to the 2nd "Getting Hired" section for my first contribution? Thank you.

**Comment by RazvanBugoi** [comment]
Can I be assigned to anything that still needs doing, please? 
Thank you.

**Comment by ManonLef** [comment]
@latsonj and @RazvanBugoi done!

**Comment by Babb2000** [comment]
I am I first-time contributor as well and would like to be assigned any open issue. Thanks.

**Comment by ManonLef** [comment]
@Babb2000 thank you @Babb2000 and I have assigned you :)

**Comment by JustWaveThings** [comment]
I only made one PR before to TOP. Can I be assigned: 
[More React Concepts](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#more-react-concepts) && [React and the Backend](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#react-and-the-backend) && [Conclusion](https://www.theodinproject.com/paths/full-stack-javascript/courses/react-new#react-and-the-backend) ?

**Comment by ManonLef** [comment]
You've got it @JustWaveThings !

**Comment by Hoslok-1** [comment]
Hey, can I do "Advanced HTML and CSS" ? I have submitted a PR for another issue in top-curriculum but I don't think it got pulled. I can get started on it after I finish my finals tomorrow. 

Thanks!

**Comment by ManonLef** [comment]
@Hoslok-1 I recently updated the headers in the Advanced HTML chapter myself so I think there will only be something left to update if I accidentally missed something. 

Would you like to do 2 sections? Maybe the Express / Mongoose one as well or are you good with this?

Good luck on your finals!

**Comment by F4OST** [comment]
can i do Express & Mongoose && Authentication && APIs 
its my first  PR

**Comment by ManonLef** [comment]
@F4OST sure! I'll add your name to it. Thank you

**Comment by Advaitmenon1106** [comment]
I don't mind taking up Javascript.

May I know by when this should be wrapped up and submitted? (Basically, is there a certain deadline?)

**Comment by Hoslok-1** [comment]
> 
Oh okay, In that case, I'll do Testing express and other two sections. 

Thank you!

**Comment by ManonLef** [comment]
@Advaitmenon1106 thanks but I offered that to Hoslok for the section they got should technically have no edits left. I will soon open up the Ruby Course as the final one to fix headings for and would love it if you join in there! 

There's no deadline but I tried to make the sections small enough so that it doesn't take too long which might be tedious to keep track off with so many people working on it.

**Comment by Advaitmenon1106** [comment]
> @Advaitmenon1106 thanks but I offered that to Hoslok for the section they got should technically have no edits left. I will soon open up the Ruby Course as the final one to fix headings for and would love it if you join in there!
> 
> There's no deadline but I tried to make the sections small enough so that it doesn't take too long which might be tedious to keep track off with so many people working on it.

Understood, and yes, I'd love to help out with the Ruby Path.

**Comment by Hoslok-1** [comment]
Hi @ManonLef , under `NodeJS`, I'm not able to find `full stack projects` folder in my local clone. Instead when I search for "where's waldo" in my local clone, it's under `javascript/javascript-and-backend`. What do I do? 

Thanks!

**Comment by ManonLef** [comment]
@Hoslok-1 If they're in that folder, I'd love it if you check them still. I am guessing they might already be updated but if not, they still have to be. It's because of the React rework changing around the projects :)

**Comment by ManonLef** [comment]
@Hoslok-1 actually, you might be looking in the Rail JS course. I believe the projects are currently to be found in the React folders in the repo.

**Comment by Hoslok-1** [comment]
> 

You were right! It was in `react`. I've just opened a PR. 

Thanks!

**Comment by ManonLef** [comment]
Hi @KadeKorn and @YamamJamal , checking in to see if you need any help with this. Just let me know if you do!

**Comment by shady54** [comment]
Hi @ManonLef ,

Is anything open,
I have never contributed to open source. Could you please assign me something.

**Comment by ManonLef** [comment]
@shady54 thanks for being willing to help out! There's currently nothing open on this issue. You can always have a look through the [issues with status `help wanted`](https://github.com/TheOdinProject/curriculum/labels/Status%3A%20Help%20Wanted) and see if any are labeled with `good first issue`.

I'll also soon be opening up this issue for the Ruby path to wrap it up.

**Comment by ManonLef** [comment]
@KadeKorn and @YamamJamal , I'd really like to start wrapping up this issue. Could you give me an indication of your progress please?

**Comment by YamamJamal** [comment]
> @KadeKorn and @YamamJamal , I'd really like to start wrapping up this issue. Could you give me an indication of your progress please?

I will have it done today, sorry for the delay!

**Comment by ManonLef** [comment]
@YamamJamal no problem! I will put a more clear time expectation on it next time as well. Working with so many people on one issue and wanting to wrap up the curriculum to avoid too many merge conflicts made this one special. Usually there's less rush to any issues you volunteer on. Thank you for doing this <3

**Comment by shady54** [comment]
@ManonLef  -- Thank you

**Comment by KadeKorn** [comment]
> @KadeKorn and @YamamJamal , I'd really like to start wrapping up this issue. Could you give me an indication of your progress please?

Hello @ManonLef, 
I believe I finished that issue. Please let me know if I did it wrong and I will be happy to re-push it.
This is my first contribution, so please let me know if I need to do it again.
Thank you,
-Kade

**Comment by ManonLef** [comment]
@KadeKorn I didn't see a PR come by for it. Did you open a PR from the fork of your curriculum repo?

**Comment by KadeKorn** [comment]
@ManonLef, I please unassign me from this. Apologies. Thank you for your time

**Comment by ManonLef** [comment]
Seems there are two sections open for someone to pick up: [Testing JS](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#testing-javascript) && [Intermediate Git](https://www.theodinproject.com/paths/full-stack-javascript/courses/javascript#intermediate-git). Just let me know if you'd like to be assigned!

**Comment by fabulousgk** [comment]
I can knock them out unless you want to save for someone new

**Comment by ManonLef** [comment]
@fabulousgk that would be great thank you! It would be nice to wrap this one up and enjoy a small break for the final path!

**Comment by ManonLef** [comment]
You did it! Thanks so much to everyone that contributed 🎉
